android/BOINC/app/src/main/java/edu/berkeley/boinc/utils/BOINCUtils.kt

```diff
@@ -21,10 +21,16 @@
 package edu.berkeley.boinc.utils
 
 import android.content.Context
+import android.net.ConnectivityManager
+import android.os.Build
 import edu.berkeley.boinc.R
 import java.io.IOException
 import java.io.Reader
 
+val ConnectivityManager.isOnline: Boolean
+    get() = if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M)
```

Comment:
What result will be in case when `Build.VERSION.SDK_INT >= Build.VERSION_CODES.M` ?

Reply:
There's an else case for that